integration/common/core.py

```diff
@@ -431,16 +431,14 @@ def reset_volume(grpc_c, *grpc_r_list):
 
 
 def create_backup(url, snap, backup_target, volume_size=SIZE_STR,
-                  backing_image_name="", backing_image_url=""):
+                  backing_image_name="", backing_image_checksum=""):
     backup = cmd.backup_create(url, snap, backup_target,
-                               [], backing_image_name, backing_image_url)
+                               [], backing_image_name, backing_image_checksum)
     backup_info = cmd.backup_inspect(url, backup)
     assert backup_info["URL"] == backup
     assert backup_info["VolumeSize"] == volume_size
     if backing_image_name != "":
         assert backup_info["VolumeBackingImageName"] == backing_image_name
-    if backing_image_url != "":
```

Comment:
Do we need to check the checksum if it's not empty?

Reply:
Now the checksum it's store in BackupVolume info only. The backup info doesn't know anything about the backing image checksum.